ms-inventory/routes/controller.go

```diff
@@ -27,47 +28,47 @@ func NewController(lc logger.LoggingClient, service interfaces.ApplicationServic
 func (c *Controller) AddAllRoutes() error {
 	var err error
 
-	err = c.service.AddRoute("/inventory", c.InventoryGet, "GET")
+	err = c.service.AddRoute("/inventory", c.InventoryGet, http.MethodGet)
 	if errWithMsg := c.errorAddRouteHandler(err); errWithMsg != nil {
 		return errWithMsg
 	}
 
-	err = c.service.AddRoute("/inventory", c.InventoryPost, "POST", "OPTIONS")
+	err = c.service.AddRoute("/inventory", c.InventoryPost, "POST", http.MethodOptions)
```

Comment:
why we have two methods, post and options?

Reply:
I am not sure @brian-intel?

Reply:
I thought `http.MethodOptions` had something to do with CORS, but now that is built in and can be removed.

Reply:
fixed per lennys response